services/QuillLMS/app/services/classlink_integration/districts_importer.rb

```diff
@@ -21,10 +19,6 @@ def run
       raise e
     end
 
-    private def allowed_districts
-      @allowed_districts ||= ALLOWLIST
-        .split(',')
-        .map(&:to_i)
-    end
+    private def allowed_classlink_instances = ClasslinkInstance.enabled
```

Comment:
I get that this method is just an alias, but it signals that only enabled instances are allowed to be processed.